### ISSUE TITLE
OAK-D-SR cannot initialize on the 0.2.0. Works on 0.1.9

### ISSUE THREAD

**Issue by Rontruong** [luxonis/depthai-viewer#60]
Okay you all scared me w/ the lack of regression tests. I feel like this sort of bug should not happen and is very easy to wrap unit tests or manual regression tests around. So I just upgraded from 0.1.9 to 0.2.0 and the viewer app is unable to connect to the camera. The error message I'm seeing is

Errror: Failed to find device after booting, error message: X_LINK_DEVICE_NOT_FOUND, Try to connect the device to a different port.

I see in the change logs that there are OAK-D-SR POE changes. Was there a regression from this?

Reverting back to 0.1.9 fixed my problem.

OS: Windows 11

**Comment by zrezke** [comment]
Intresting. For me it works fine on 0.2.0:

![image](https://github.com/luxonis/depthai-viewer/assets/59307111/4338d41f-c3e3-4782-97c4-a2860a6705ea)

Could you post the output of [calibration_dump.py](https://github.com/luxonis/depthai-python/blob/main/examples/calibration/calibration_dump.py)


```sh

git clone https://github.com/luxonis/depthai-python

cd depthai-python

python examples/install_requirements.py

python examples/calibration/calibration_dump.py

```

Then copy paste the output, or put it in a file. It could be that you have a different board revision and maybe it's causing the issue.

**Comment by Rontruong** [comment]
Okay, this is no longer a problem. I don't know which part fixed it
- I downgraded the standalone to 0.1.9 and it worked
- After 1.5 months, it started crashing on init, bad hash or something? Reinstalling standalone did not fix it.
`oak-d Current commit: c0a2d6ea178eb3b88ee9361c027280aa571d1ef4 
Unknown error occured (Invalid version: 'latest'), exiting...`

- I gave up w/ the standalone and installed on my base python w/ python -m pip install  depthai-viewer (0.2.4). This works.
